template/app/buy-me-coffee/DisableSSR.tsx

```diff
@@ -0,0 +1,14 @@
+'use client';
+
+import { PropsWithChildren, useEffect, useState } from 'react';
+
+export function DisableSSR({ children }: PropsWithChildren) {
```

Comment:
nit. add some comments just to have it a bit more clear understanding the reason for this component

Reply:
I think this repo already has it's own convention of doing it using nextjs dynamic import with ssr: false